second-gen/packages/swc/components/progress-circle/ProgressCircle.ts

```diff
@@ -11,12 +11,18 @@
  */
 
 import { CSSResultArray, html, TemplateResult } from 'lit';
-import { ifDefined } from 'lit/directives/if-defined.js';
+import { classMap } from 'lit/directives/class-map.js';
 
 import { ProgressCircleBase } from '@swc/core/components/progress-circle';
 
 import progressCircleStyles from './progress-circle.css';
 
+function capitalize(str?: string): string {
```

Comment:
I kind of liked this utility living separately but let me know how you want to handle this in the SWC infrastructure.